docs/content/querying/segmentmetadataquery.md

```diff
@@ -5,6 +5,7 @@ layout: doc_page
 Segment metadata queries return per segment information about:
```

Comment:
since you have this open, please make this say "return per-segment information"?